src/components/ArtifactCard.vue

```diff
@@ -132,6 +132,8 @@
 .artifact-card__body { @apply
   flex
   flex-col
+  h-full
```

Comment:
I don't remember exactly what css this is but why set the height on this component? Usually a component like this would have height auto